classes/publicationFormat/PublicationFormat.inc.php

```diff
@@ -517,6 +517,25 @@ function _checkRequiredFieldsAssigned() {
 	function getPressId() {
 		return $this->getContextId();
 	}
+
+	/**
+	 * Return the format's physical dimensions
+	 * @return string
+	 */
+	function getDimensions() {
+
+		if (!$this->getPhysicalFormat()) {
+			return '';
+		}
+
+		$dimensions = array(
+			$this->getWidth() . $this->getWidthUnitCode(),
+			$this->getHeight() . $this->getHeightUnitCode(),
+			$this->getThickness() . $this->getThicknessUnitCode(),
+		);
+
+		return join( __('notification.type.visitCatalog'), $dimensions );
```

Comment:
Oh -- I see how it's being applied. I'd suggest moving the variables into the locale key itself, so they can be formatted as desired.